client/components/PetTypeTile.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+const PetTypeTile = (props) => {
+    const { id, type, description, imgUrl } = props.petType
+    return (
+        <div>
+            <div>
+                <img src={imgUrl} />
+            </div>
+            <h3>
+                {type}
+            </h3>
+            {description}

```

Comment:
might want to put the description in HTML tags, like the `type` is within an `h3`